### ISSUE TITLE
startRenderLoop does not work if called while in XR mode

### ISSUE THREAD

**Issue by ryantrem** [BabylonJS/BabylonReactNative#87]
This is a regression as it was not an issue before the alpha.19 release. I suspect this is related to the `Graphics` related changes.

If you "suspend" (e.g. go to the app switcher) when in XR mode, then "resume" (e.g. select the app from the app switcher), rendering does not continue. On Android you just get a black screen and on iOS you just get the last frame rendered. If you exit XR mode from there, the app crashes. If I comment out the code that stops the render loop on app pause and starts the render loop when the app becomes active again, then this issue does not repro.

This doesn't repro in the Babylon Native Playground app, but can be reproed by stopping and starting the render loop while in XR mode. E.g. update `experience.js` as follows:
```typescript
xr.baseExperience.enterXRAsync(vr ? "immersive-vr" : "immersive-ar", "unbounded", xr.renderTarget).then(() => {
    return new Promise(resolve => setTimeout(resolve, 3000));
}).then(() => {
    console.log("STOPPING THE RENDER LOOP");
    engine.stopRenderLoop();
}).then(() => {
    return new Promise(resolve => setTimeout(resolve, 2000));
}).then(() => {
    console.log("STARTING THE RENDER LOOP");
    engine.runRenderLoop(function () {
        scene.render();
    });
});
```

**Comment by ryantrem** [comment]
It turns out you don't even need to "restart" the render loop. If you just go into XR mode *before* starting the render loop, and then call `startRenderLoop`, nothing gets rendered (and I don't think `RequestAnimationFrame` is even called, but I need to confirm).

**Comment by syntheticmagus** [comment]
Addressed by https://github.com/BabylonJS/BabylonNative/pull/434.

**Comment by syntheticmagus** [comment]
Should be fixed now that BabylonJS/BabylonNative#434 has gone in.
